frontend/components/ClubPage/Events.js

```diff
@@ -0,0 +1,53 @@
+import { Icon, Text, StrongText, Card } from '../common'
+import { useState, useEffect } from 'react'
+
+const Events = props => {
+  const data = props.data
+
+  const iconStyles = {
+    float: "left",
+    marginRight: "7px"
+  }
```

Comment:
Generally don't want float, would be good if we could do the same thing with flexbox or `margin: auto`

Reply:
I think I got this resolved using flexboxes. Thanks for the heads up